core/bayesian/utils.py

```
@@ -0,0 +1,274 @@
+EN: Helper functions for Bayesian computations.
+
+Provides:
+- Anomaly scoring (z-score, probability-based)
+- Confidence level classification
+- Statistical testing utilities
+- Prior generation from data
+"""
+
+from __future__ import annotations
+
+from typing import List, Optional, Tuple
```

Flake8 (7.3.0)

[error] 'typing.Optional' imported but unused

(F401)